src/env.d.ts

```diff
@@ -1 +1,2 @@
+/// <reference path="../.astro/types.d.ts" />
```

Comment:
???

Reply:
This line was autogenerated when 'prefetch' dependency was added